### ISSUE TITLE
Cannot read property 'uri' of undefined

### ISSUE THREAD

**Issue by dapperdandev** [romanresh/vscode-testcafe#20]
Possible issue with extension. I get this anytime I try to run a test via right-click test or fixture, then select
**TestCafe: Run Test(s) in Chrome**. This worked fine until about a week ago.

TestCafe Test Runner: 1.4.0
VSCode: 1.17.2
Windows 10 Pro (1703) / OS Build 15063.674

![image](https://user-images.githubusercontent.com/13296622/32018459-2aaffc90-b987-11e7-8566-e18049cfb31a.png)

**launch.json**
```javascript
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "type": "node",
            "request": "launch",
            "name": "Launch Program",
            "program": "${workspaceFolder}/tests\\Web\\Release\\release.test.ts",
            "outFiles": [
                "${workspaceFolder}/**/*.js"
            ]
        }
    ]
}
```

**Comment by AlexanderMoskovkin** [comment]
As I know it's a bug in VSCode. They already fixed it but it's not included in the last release yet. I guess @romanresh will tell us details about this tomorrow and probaply give a workaround.

**Comment by dapperdandev** [comment]
Thanks @AlexanderMoskovkin!

**Comment by romanresh** [comment]
Hi @djbreen7 . Do your tests run correctly after this error appears? Because, in my environment the error appears in VS but it doesn't affect on test execution.

This bug is fixed in vscode in the context of this issue: https://github.com/Microsoft/vscode/issues/36392. I think, the fix will be available in the November's update of VS Code (with coming back to the blue color of the vscode's icon).

**Comment by dapperdandev** [comment]
@romanresh: The tests never run, unfortunately. It immediately kills the process.

**Comment by dapperdandev** [comment]
@romanresh: ![covfefe](https://user-images.githubusercontent.com/13296622/32117734-9c563180-bb0c-11e7-95b0-ccccb0a51a87.gif)

**Comment by romanresh** [comment]
Fixed in v1.4.1.
